cms/envs/common.py

```diff
@@ -793,6 +793,7 @@
 
 OPTIONAL_APPS = (
     'mentoring',
+    'problem_builder',
```

Comment:
I don't think it matters, because all we need is for the tables to be created, and the cms and lms migrations are usually run together as part of `paver update_db`. But since the old mentoring was listed in both, and could potentially be important in the future, I might as well add it in there too.